h2/src/main/org/h2/command/ddl/DropView.java

```diff
@@ -67,9 +68,26 @@ public int update() {
                     }
                 }
             }
+            
+             // TODO: Where is the ConstraintReferential.CASCADE style drop processing ? It's
+            // supported from imported keys - but not for dependent db objects            
 
+            TableView tableView = (TableView) view;
+            ArrayList<Table> copyOfDependencies = new ArrayList<Table>(tableView.getTables());
+            
             view.lock(session, true, true);
             session.getDatabase().removeSchemaObject(session, view);
+            session.getDatabase().unlockMeta(session);
```

Comment:
move the unlock to below the removal of the dependant table expressions

Reply:
Done